src/main/java/hudson/plugins/matrix_configuration_parameter/DefaultMatrixCombinationsParameterValue.java

```diff
@@ -75,6 +75,9 @@ public boolean combinationExists(AxisList axes, Combination c){
     
     @Override
     public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }

```

Comment:
<BugInstance type='NP_EQUALS_SHOULD_HANDLE_NULL_ARGUMENT' priority='Normal' category='BAD_PRACTICE' message='hudson.plugins.matrix_configuration_parameter.DefaultMatrixCombinationsParameterValue.equals(Object) does not check for null argument' lineNumber='78'/>